### PR TITLE
Add Naver Provider

### PR DESCRIPTION
**Pull request by christallire**
Adding South Korea's dominating search engine provider, Naver.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -162,6 +163,7 @@ If a provider you're looking for does not exist, consider making a PR to add one
...
 | Myob | [![NuGet](https://buildstats.info/nuget/AspNet.Security.OAuth.Myob?includePreReleases=false)](https://www.nuget.org/packages/AspNet.Security.OAuth.Myob/ "Download AspNet.Security.OAuth.Myob from NuGet.org") | [![MyGet](https://buildstats.info/myget/aspnet-contrib/AspNet.Security.OAuth.Myob?includePreReleases=true)](https://www.myget.org/feed/aspnet-contrib/package/nuget/AspNet.Security.OAuth.Myob "Download AspNet.Security.OAuth.Myob from MyGet.org") | [Documentation](https://developer.myob.com/api/accountright/api-overview/authentication/ "Myob developer documentation") |
+| Naver | [![NuGet](https://buildstats.info/nuget/AspNet.Security.OAuth.Naver?includePreReleases=false)](https://www.nuget.org/packages/AspNet.Security.OAuth.Naver/ "Download AspNet.Security.OAuth.Naver from NuGet.org") |  [![MyGet](https://buildstats.info/myget/aspnet-contrib/AspNet.Security.OAuth.Naver?includePreReleases=true)](https://www.myget.org/feed/aspnet-contrib/package/nuget/AspNet.Security.OAuth.Naver "Download AspNet.Security.OAuth.Naver from MyGet.org") |  [Documentation](https://developers.naver.com/docs/login/api/api.md "Naver login api developer documentation") |
```
Comment (martincostello) [inline README.md]:
```suggestion
| Naver | [![NuGet](https://buildstats.info/nuget/AspNet.Security.OAuth.Naver?includePreReleases=false)](https://www.nuget.org/packages/AspNet.Security.OAuth.Naver/ "Download AspNet.Security.OAuth.Naver from NuGet.org") |  [![MyGet](https://buildstats.info/myget/aspnet-contrib/AspNet.Security.OAuth.Naver?includePreReleases=true)](https://www.myget.org/feed/aspnet-contrib/package/nuget/AspNet.Security.OAuth.Naver "Download AspNet.Security.OAuth.Naver from MyGet.org") |  [Documentation](https://developers.naver.com/docs/login/api/api.md "Naver login API developer documentation") |
```

File: src/AspNet.Security.OAuth.Naver/AspNet.Security.OAuth.Naver.csproj [hunk 2]
```diff
@@ -0,0 +1,18 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>$(DefaultNetCoreTargetFramework)</TargetFrameworks>
+  </PropertyGroup>
```
Comment (martincostello) [inline src/AspNet.Security.OAuth.Naver/AspNet.Security.OAuth.Naver.csproj]:
```suggestion
  <PropertyGroup>
    <PackageValidationBaselineVersion>6.0.5</PackageValidationBaselineVersion>
    <TargetFrameworks>$(DefaultNetCoreTargetFramework)</TargetFrameworks>
  </PropertyGroup>

  <!-- TODO Remove once published to NuGet.org -->
  <PropertyGroup>
    <DisablePackageBaselineValidation>true</DisablePackageBaselineValidation>
  </PropertyGroup>
```

**Review by martincostello — APPROVED** [review]

File: src/AspNet.Security.OAuth.Naver/AspNet.Security.OAuth.Naver.csproj (hunk 2 above)
Comment (martincostello) [reply src/AspNet.Security.OAuth.Naver/AspNet.Security.OAuth.Naver.csproj]: I was just going to commit these to the PR myself and merge it, but I'm not being given permission to do so, so I'm guessing you didn't enable to option when you opened the PR.

This change is required for the CI to pass.

**Comment by christallire** [comment]
Both committed, thanks for the thorough review.

**Comment by martincostello** [comment]
Just to confirm, you've validated the new provider against the real Naver service?

**Comment by christallire** [comment]
> Just to confirm, you've validated the new provider against the real Naver service?

Yeah, definitely. I'm using it for the development environment and waiting for the package to deploy it to prod.

**Comment by martincostello** [comment]
Great, thanks. I'll look at getting this deployed to NuGet today. Once it's merged you'll be able to use a prerelease package from [our MyGet feed](https://www.myget.org/gallery/aspnet-contrib) until that happens.

**Comment by martincostello** [comment]
[MyGet package](https://www.myget.org/feed/aspnet-contrib/package/nuget/AspNet.Security.OAuth.Naver/6.0.5-preview.22177.36)

**Comment by martincostello** [comment]
Thanks again for your contribution @christallire - the Naver package is now available from [NuGet.org](https://www.nuget.org/packages/AspNet.Security.OAuth.Naver/) 🎈

**Pull request merged**